src/resources/MachineLearning/ModelDetailedInfo/details/ModelDetailsStatsPerSource.ts

```diff
@@ -4,4 +4,8 @@ export interface ModelDetailsStatsPerSource {
     invalidHtmlDocumentCount: number;
     documentWithoutIdCount: number;
     documentWithDuplicatedIdCount: number;
+    documentsWithChunksCount?: number;
+    documentsWithChunksRatio?: number;
+    chunkCount?: number;
+    chunksPerDocument?: number;
```

Comment:
Just double checking, is this one really a number?

```suggestion
    chunksPerDocument?: {
        min?: number;
        max?: number;
        mean?: number;
    };
```

Reply:
no it's not good catch :facepalm: I even wrote so in the table and I confused it with the other one